src/metatrain/experimental/soap_bpnn/trainer.py

```diff
@@ -205,7 +205,11 @@ def train(
         logging.info(f"Training with loss weights: {loss_weights_dict_external}")
 
         # Create a loss function:
-        loss_fn = TensorMapDictLoss(loss_weights_dict)
+        loss_fn = TensorMapDictLoss(
+            loss_weights_dict,
+            reduction=self.hypers["loss"]["reduction"],
+            type=self.hypers["loss"]["type"],
```

Comment:
Does it make sens to design the function is such a way that you simply pass

```python
loss_fn = TensorMapDictLoss(**self.hypers["loss"])
```

Reply:
Good idea, I'll try to do it